### ISSUE TITLE
Sign in fields have no labels

### ISSUE THREAD

**Issue by Rakielle** [uyuni-project/uyuni#6775]
# Short description
The sign in fields of Uyuni has no labels. Adding labels to the fields would improve accessibility.

# Details
As part of openSUSE/mentoring#190, this issue hopes to improve on the accessibility for the sign in page and one feature that could be improved is the sign in form. Currently, the sign in form has no labels for the fields.
<img width="359" alt="Screenshot 2023-03-23 at 22 45 41" src="https://user-images.githubusercontent.com/48498778/227374346-d79e5e78-5691-4ddd-959c-65d63c1e93c6.png">
 This is an accessibility issue because if a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users.

To fix this we could add a label element for each of the sign in fields.

# References
[Non-text Content](https://webaim.org/standards/wcag/checklist#sc1.1.1)
[Info and Relationships](https://webaim.org/standards/wcag/checklist#sc1.3.1)
[Headings and Labels](https://webaim.org/standards/wcag/checklist#sc2.4.6)
[Labels or Instructions](https://webaim.org/standards/wcag/checklist#sc3.3.2)

**Comment by deneb-alpha** [comment]
Thanks for reporting this @Rakielle :)

I assigned the issue to you, feel free to start to work at a PR for a fix.
